consensus/hotstuff/signature.go

```diff
@@ -0,0 +1,60 @@
+package hotstuff
+
+import (
+	"github.com/onflow/flow-go/crypto"
+	"github.com/onflow/flow-go/model/flow"
+)
+
+// RandomBeaconReconstructor collects verified signature shares, and reconstructs the
+// group signature with enough shares.
+type RandomBeaconReconstructor interface {
```

Comment:
Very confusing, what is the difference between `RandomBeaconReconstructor` and `RandomBeaconAggregator` ?